### PR TITLE
[rhwa] rapidast container definition

### PR DESCRIPTION
**Pull request by frajamomo**
Dockerfile that provides the capability of running rapidast test from eco-gotest code.

### CODE REVIEW

**Comment by frajamomo** [comment]
@kononovn please review

File: images/rhwa/dast/config/rapidastConfig.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    #  - See https://aquasecurity.github.io/trivy/v0.49/docs/target/kubernetes/ for more information on 'trivy k8s' scan
+    #  - scanners/generic/tools/convert_trivy_k8s_to_sarif.py converts the Trivy json result to the SARIF format
+    # 'inline' is used when container.type is not 'podman'
+    # 'toolDir' specifies the default directory where inline scripts are located
+    #toolDir: scanners/generic/tools
+    inline: "trivy k8s -n $NAMESPACE pod --severity=HIGH,CRITICAL --scanners=misconfig --report all --format json"
+    container:
+      parameters:
+        # Optional: list of expected return codes, anything else will be considered as an error. by default: [0]
+        validReturns: [ 0 ]
```
Comment (kononovn) [inline images/rhwa/dast/config/rapidastConfig.yaml:46]: Can you add empty line to the end of the file ?

**Review by kononovn — CHANGES_REQUESTED** [review]
1 small comment

**Review by kononovn — APPROVED** [review]
/lgtm

**Pull request merged**